### PR TITLE
Prepare Repo for Labels

### PR DESCRIPTION
**Pull request by werner-roman**
Prepare repo to contain the labels, contract and a readme with details about the process

**Tasks:**
- [X] add readme
- [X] add labels
- [x] add contract
- [x] update links in readme

### CODE REVIEW

**Review by werner-roman — COMMENTED** [review]
Standard Subject: "Catena-X - Logo Use Agreement"

**Comment by werner-roman** [comment]
@AnnKaCatX  - FYI

**Comment by werner-roman** [comment]
Hey @AnnKaCatX , @stephanbcbauer , @jSchuetz88 ,

honestly not sure if we have mores tuff if we should put it in the library itself - as of now I would say its "ok"?

**Review by AnnKaCatX — APPROVED** [review]

**Comment by werner-roman** [comment]
TODO: Labels need to be updated from "Qualified Advisory" to "Qualified Advisor"

File: README.md [hunk 1]
```diff
@@ -1 +1,21 @@
...
\ No newline at end of file
+# Catena-X Resources
+
+This repository offers various Catena-X contents to download.
+
+## Use of Catena-X Labels
+
+To use the labels, you **must**:
+
+1. Downloaded the contract [HERE](./static/Logo%20Use%20Agreement_V2_BOtOG.pdf)
```
Comment (HFocken) [inline README.md]: I would change "contract" to "Logo Use Agreement" - it makes it more specific and matches the document.

**Review by HFocken — CHANGES_REQUESTED** [review]
LGTM after one small change!

File: README.md (hunk 1 above)
Comment (werner-roman) [reply README.md]: Changed it everywhere :)

**Review by HFocken — APPROVED** [review]
Thank you for incorporating the requested changes - LGTM!

**Pull request merged**